src/main/resources/messages.properties

```diff
@@ -37,44 +37,23 @@ language-preferences.english=English
-index.immigration-status-examples=A government-issued document such as a Visa, work permission, or Green Card.
-index.income=<b>Proof of work income</b>
-index.income-examples=Tax documents from 2021 or 2022 (such as Form 1040), or wage documents (such as pay stubs, W-2)  for all adults in the household that earn work income.
-index.expenses=<b>Proof of money spent on certain services</b>
-index.expenses-examples=Rent or mortgage, utility, childcare, or medical bills.
-index.immunization-records=<b>For FITAP applications: Proof of immunization</b>
-index.immunization-record-examples=Photocopies of immunization records from a licensed healthcare provider or clinic.
-index.non-work-income=<b>Proof of non-work income</b>
-index.non-work-income-examples=Proof of income letters for non-work income such as Social Security or Supplemental Security Income (SSI) for all adults in the household that earn non-work income.
-index.food-programs=<b>Food assistance programs</b>
-index.food-programs-l1=<b>SNAP</b> (Supplemental Nutrition Assistance Program, formerly Food Stamps)
-index.food-programs-l2=<b>LaCAP</b> (Louisiana Combined Application Project)
-index.cash-programs=<b>Cash assistance programs</b>
-index.cash-programs-l1=<b>FITAP</b> (Family Independence Temporary Assistance Program)
-index.cash-programs-l2=<b>KCSP</b> (Kinship Care Subsidy Program)
-index.snap.description=<b>SNAP</b> (Supplemental Nutrition Assistance Program) helps you and your family get the food they need. You'll get an EBT card that you can use like a debit card to buy groceries at most stores and farmer's markets.
-index.lacap.description=<b>LaCAP</b> helps Louisianans who are at least 60 years of age, and receive Supplemental Security Income (SSI), get the food they need. It is a simplified version of SNAP.
-index.fitap.description=<b>FITAP</b> (Family Independence Temporary Assistance Program) provides cash assistance to help people meet their basic needs until they are able to support themselves.
-index.kscp.description=<b>KCSP</b> (Kinship Care Subsidy Program) also provides cash assistance to relatives who are taking care of children who are unable to live with their parents. Eligible relatives can receive monthly cash payments to help cover the costs of caring for the child, such as housing, food, clothing, and other basic needs.
-index.looking-for-help=<b>Looking for assistance with other programs?</b>
-index.visit-dcfs-website=Visit <a target="_blank" rel="noreferrer noopener" href="https://www.dcfs.louisiana.gov">the Department of Children and Family Services website</a> for more information.
+index.upload-documents-second=Aplique ya<i class="button__icon icon-arrow_forward"></i>
+index.upload-documents-second.link=/flow/laDigitalAssister/parish?lang=es
+index.upload-documents-third=\u0110\u0103ng k\u00FD ngay<i class="button__icon icon-arrow_forward"></i>
+index.upload-documents-third.link=/flow/laDigitalAssister/parish?lang=vi
```

Comment:
just fyi - these are the properties re Karens comment on the Asana ticket. Just need to update them in the messages_es and messages_vi property files too

Reply:
Actually - might be better to change it in transifex

Reply:
Thanks for flagging these sree. I went ahead and fixed them